designs/teagan/src/front.mjs

```diff
@@ -181,6 +231,8 @@ export const front = {
     backNeckCutout: { pct: 8, min: 4, max: 12, menu: 'fit' },
     // Teagan specific
     draftForHighBust: { bool: false, menu: 'fit' },
+    curveToWaist: { bool: false, menu: 'fit' },
+    waistEase: { pct: 25, min: 8, max: 40, menu: 'fit' },
```

Comment:
You might want to consider renaming the `waistEase` option name to something like `curvedWaistEase` to make it more obvious that it affects only curved waists. The documentation and translation will help with this, but if the translation string goes untranslated then changing the name would help avoid confusion.

(Or, you could change the code to always use `waistEase` in the width calculations.)

Reply:
That's a good idea! Done.